### PR TITLE
Docker - use main instead of master branch (only to be merged when 2.16 is about to be merged into main)

### PR DESCRIPTION
**Pull request by taniwallach**
Modify the build args section:
  - To use main branches by default.
  - Layout change of the settings.

Add a minor comment regarding ADD_APT_PACKAGES and startup speed.

**Should be merged only shortly before 2.16 is merged into main.**

### CODE REVIEW

**Review by drgrice1 — APPROVED** [review]
This generally looks good.  However, should the defaults in the WeBWorK 2.16 branch be for the WeBWorK-2.16 and PG-2.16 branches?  And then switch that to main right before the official merge?

**Comment by taniwallach** [comment]
> However, should the defaults in the WeBWorK 2.16 branch be for the WeBWorK-2.16 and PG-2.16 branches? And then switch that to main right before the official merge?

Yes. My intention was for this to only be merged in just before 2.16 gets merged into main, as which point the 2.16 branch is only of historical interest. That was what I intended by the line at the end of the first post.

File: docker-compose.yml [hunk 1]
```diff
@@ -77,15 +77,22 @@ services:
...
+            # Exactly one pair of settings should be left uncommented and all the others
+            # should be commented out.
+
+            # main branches:
+            - WEBWORK2_BRANCH=main
+            - PG_BRANCH=main
+
+            # WeBWorK/PG 2.16 branches:
+            #- WEBWORK2_BRANCH=WeBWorK-2.16
+            #- PG_BRANCH=PG-2.16
```
Comment (drgrice1) [inline docker-compose.yml:95]: All of the 2.16 references should not be here.  Instead these should be references to the develop branch.  After a while the 2.16 branches will be deleted.

Comment (taniwallach) [reply docker-compose.yml]: Good point. It was meant as a sample alternate option, but using develop is a better idea.
Done.

**Review by drgrice1 — APPROVED** [review]
Looks good.  This will be merged immediately before the WeBWorK-2.16 branch is merged into main.

**Pull request merged**